src/structures/interfaces/TextBasedChannel.js

```diff
@@ -204,14 +195,9 @@ class TextBasedChannel {
    * @returns {Promise<Collection<Snowflake, Message>>}
    */
   fetchPinnedMessages() {
```

Comment:
If `fetchMessage` and `fetchMessages` are meant to be moved to `MessageStore` this method could also be moved there.